dll/ntdll/ldr/ldrapi.c

```diff
@@ -781,7 +742,7 @@ LdrGetDllHandleEx(
     }
 
     /* Free string if needed */
-    if (DllString1.Buffer) RtlFreeUnicodeString(&DllString1);
+    if (DynamicString.Buffer) RtlFreeUnicodeString(&DynamicString);
```

Comment:
put the rtlfree in separate line (as you did for another one above)

Reply:
I am not going to reformat all code.